extensions/amp-a4a/0.1/template-validator.js

```diff
@@ -65,21 +68,26 @@ export class TemplateValidator extends Validator {
       .fetch(parsedResponseBody.templateUrl)
       .then((template) => {
         const creativeMetadata = getAmpAdMetadata(template);
-        const customElementExtensions =
-          creativeMetadata['customElementExtensions'];
-        if (parsedResponseBody.analytics) {
-          pushIfNotExist(customElementExtensions, 'amp-analytics');
+        creativeMetadata['extensions'] = creativeMetadata['extensions'] || [];
+        const extensions = creativeMetadata['extensions'];
+        if (
+          parsedResponseBody.analytics &&
+          !extensionsHasElement(extensions, 'amp-analytics')
+        ) {
+          extensions.push({
+            'custom-element': 'amp-analytics',
+            src: 'https://cdn.ampproject.org/v0/amp-analytics-0.1.js',
+          });
+        }
+        if (!extensionsHasElement(extensions, 'amp-mustache')) {
+          extensions.push({
+            'custom-element': 'amp-mustache',
+            src: 'https://cdn.ampproject.org/v0/amp-mustache-0.1.js',
```

Comment:
This should normally be `0.2` or maybe better "latest". This is for mustache specifically.